phnt/include/ntzwapi.h

```diff
@@ -326,7 +326,7 @@ ZwAllocateUserPhysicalPagesEx(
     _In_ HANDLE ProcessHandle,
     _Inout_ PULONG_PTR NumberOfPages,
     _Out_writes_(*NumberOfPages) PULONG_PTR UserPfnArray,
-    _Inout_updates_opt_(ParameterCount) PMEM_EXTENDED_PARAMETER ExtendedParameters,
+    _Inout_updates_opt_(ExtendedParameterCount) PMEM_EXTENDED_PARAMETER ExtendedParameters,
```

Comment:
https://github.com/winsiderss/systeminformer/blob/c69571882b479316a157cfc16c03b58ecdabd267/phnt/include/ntzwapi.h#L3-L5

Reply:
Committed, this file is reverted and untouched.